general/cisetup/cisetup.go

```diff
@@ -56,6 +56,7 @@ const (
 	ideUserPassPlaceholder   = "<INSERT-PASSWORD>"
 	ideUserEmailPlaceholder  = "<INSERT-EMAIL>"
 	createUserTemplate       = `jfrog rt user-create "%s" "%s" "%s" --users-groups="%s" --server-id="%s"`
+	iterationsTimeout        = 200
```

Comment:
Since this variable doesn't indicate a timeout, but attemprts, I suggest renaming it to "maxRepoCreationAttempts".